spec/binarySearchTree.js

```diff
@@ -0,0 +1,98 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import BinarySearchTree from '../src/BinarySearchTree'
+import BinaryNode from '../src/binarynode'
+
+chai.use(chaiChange)
+
+describe('BinarySearchTree', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(BinarySearchTree).to.be.a('function')
+  })
+
+  context('insert()', () => {
+    context('when the tree is empty', () => {
+      it('inserts a node with the specified value at the root', () => {
+          const myBST = new BinarySearchTree()
```

Comment:
This is totally a nitpicky me thing - I dislike identifiers that begin with _my_.  This is just an instance of a `BinarySearchTree`, it doesn't need ownership. 😄   I would probably name it simply `bst`.